app/routes/PostTravelProductRoute.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const postTravelProduct = require('../controllers/PostTravelProductController');
+module.exports = function (app) {
+
+    app.route('/postTravelProduct/byId/:postTravelProductId')
+        .get(postTravelProduct.show)
+        .put(postTravelProduct.updateProduct);
+
+    app.route('/postTravelProduct')
+        .get(postTravelProduct.all)
+        .post(postTravelProduct.createProduct, postTravelProduct.createProductDocument);
+
+    app.route('/postTravelProduct/byProductCategoryId/:productCategoryId')
+        .get(postTravelProduct.show);
+
+    app.route('/postTravelProduct/:userProfileId')
```

Comment:
app.route('/posttravelproduct/:profileId')

Reply:
No need to change to userProfileId as profileId can be use.

Reply:
Cannot have the same api name in the project